app/src/androidTest/java/com/github/factotum_sdp/factotum/utils/ContactsUtils.kt

```diff
@@ -3,14 +3,82 @@ package com.github.factotum_sdp.factotum.utils
 import android.widget.TextView
 import androidx.recyclerview.widget.RecyclerView
 import com.github.factotum_sdp.factotum.R
+import com.github.factotum_sdp.factotum.placeholder.Contact
 import com.google.firebase.database.FirebaseDatabase
+import com.google.firebase.database.ktx.database
+import com.google.firebase.ktx.Firebase
+import kotlinx.coroutines.CompletableDeferred
 import org.hamcrest.Description
 import org.hamcrest.Matcher
 import org.hamcrest.TypeSafeMatcher
 
 class ContactsUtils {
+
     companion object {
-        fun emptyFirebaseDatabase(database: FirebaseDatabase) {
+        //fake data --> to be replaced with connection to database
+        private val randomContacts = mutableListOf<Contact>()
+        private var emulatorSet : Boolean = false
+        private var database : FirebaseDatabase = Firebase.database
+        private val randomNames = listOf("John", "Jane", "Joe")
+        private var randomSurnames = listOf("Smith", "Jones", "Williams")
+        private val roles = listOf("Boss", "Courier", "Client")
+        private val randomAddresses = listOf("123 Fake Street", "456 Fake Street", "789 Fake Street")
+        private val randomPhones = listOf("123456789", "987654321", "123987456")
+        private val randomDetails = listOf("I am a boss", "I am a courier", "I am a client")
+
+        private const val image = R.drawable.contact_image
+
+        fun setEmulatorGet() : FirebaseDatabase {
```

Comment:
I don't think this method should be linked to the Contacts, it's a very general method that is used in almost every test class.